www/src/pages/faq.md

```diff
@@ -4,11 +4,130 @@ pageTitle: FAQ
+
+## How do I get involved?
+
+Join the official [W3C Design Tokens Community Group](https://www.w3.org/community/design-tokens/): Participate in formal specification development discussions, attend group meetings and calls, contribute to the working drafts and proposals, and influence the direction of the specification.
+
+Learn more on how to [contribute](../contribute/).
+
+## What tools support this?
+
+Organizations building design tools and open-source projects are already shipping DTCG-compatible tokens. Here are some of the adopters and participants helping prove out the spec:
+
+- [Abstract](https://abstract.com)
+- [Adobe](https://adobe.com)
+- [Project Wallace](https://www.github.com/projectwallace/css-design-tokens/)
+- [Design Token Validator (Anima)](https://www.animaapp.github.io/design-token-validator-site/)
+- [Design Tokens Language Server](https://www.github.com/bennypowers/design-tokens-language-server)
+- [Figma](https://www.figma.com)
+- [Framer](https://www.framer.com)
+- [Knapsack](https://www.knapsack.cloud)
+- [Marvel](https://www.marvelapp.com)
+- [Penpot](https://www.penpot.com)
+- [Sketch](https://www.sketch.com)
+- [Style Dictionary](https://www.styledictionary.com)
+- [Superposition](https://www.superposition.design)
+- [Supernova](https://www.supernova.ion)
+- [Terrazzo](https://www.terrazzo.app)
+- [Toolabs](https://www.toolabs.com)
+- [TokensBrücke](https://www.figma.com/community/plugin/1254538877056388290)
+- [Tokens Studio](https://www.tokens.studio)
+- [Universal Design Tokens](https://www.github.com/universal-design-tokens/udt)
+- [Zeplin](https://www.zeplin.io)
+- [zeroheight](https://www.zeroheight.com)
+
+## How does this relate to other token approaches?
+
+The Design Tokens Community Group specification relates to other token approaches as a standardization layer that unifies previously fragmented, tool-specific formats. It provides:
+
+- **Canonical syntax** replacing incompatible proprietary formats
+- **Interoperability foundation** enabling cross-tool workflows
+- **Validation framework** ensuring format consistency
+- **Extension mechanism** preserving tool-specific capabilities
+
+The specification **complements rather than replaces** token methodologies and architectural patterns developed by production design systems. It defines the format for token exchange while leaving organizational strategy to design system teams.
+
+As of 2025, the specification approaches v1.0.0 release with growing adoption across major design tools and transformation pipelines. Teams should evaluate DTCG migration based on interoperability requirements, tooling support, and long-term maintenance considerations.
+
+## What is the relationship between JSON Schema and DTCG?
+
+JSON Schema is a declarative specification, a passive validator, and documentation format. It can be used to enforce documents to adhere to a strict format. It is a general purpose data interchange format.
```

Comment:
I think this still has some remnants of confusing JSON and JSON Schema. JSON Schema is not a general purpose data interchange format. It’s not quite accurate to say it’s a validator, either; technically those are JSON Schema tools. 

```suggestion
[JSON Schema](https://json-schema.org/) is a tool for describing JSON structures.
```